src/backend/renderer/element/mod.rs

```diff
@@ -89,6 +96,50 @@ pub enum UnderlyingStorage<'a, R: Renderer> {
     External(&'a R::TextureId),
 }
 
+/// Defines the possible states of an element
+/// after rendering
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum RenderElementState {
+    /// The element was rendered
+    Rendered,
+    /// The element was directly scanned out
+    ScannedOut,
```

Comment:
Ideally this would perhaps recorder the presented time. Perhaps `ScannedOut(Duration)`

Reply:
Okay, totally my fault, this is named pretty badly. That only means it has been assigned to a plane for scan out.